README.md

```diff
@@ -127,23 +127,44 @@ DSPACE_REST_NAMESPACE # The namespace of the REST application
 DSPACE_REST_SSL # Whether the angular REST uses SSL [true/false]
 ```
 
+All other settings can be set using the following convention for naming the environment variables:
+
+1. replace all `.` with `_`
+2. convert all characters to upper case
+
+e.g.
+
+```
+cache.msToLive.default => CACHE_MSTOLIVE_DEFAULT
+auth.ui.timeUntilIdle => AUTH_UI_TIMEUNTILIDLE
```

Comment:
Could we consider changing this convention slightly to ensure all environment variables start with `DSPACE_` (or alternatively `DS_`)?  I'm a little worried we could accidentally have an env variable conflict/collision if we name a configuration similar to a non-DSpace specific environment variable.  For example, if we ever had a setting named "home" or "path", those would conflict with the `HOME` and `PATH` standard env variables.

So, I'd change this mapping to be something like..
```
cache.msToLive.default => DSPACE_CACHE_MSTOLIVE_DEFAULT
auth.ui.timeUntilIdle => DSPACE_AUTH_UI_TIMEUNTILIDLE
```

Reply:
No argument there. Makes perfect sense. Will work on it...

Reply:
Used prefix `DSPACE`.

Reply:
@wwelling : Tiny thing, but you forgot to update these two examples in the README:
```
cache.msToLive.default => CACHE_MSTOLIVE_DEFAULT
auth.ui.timeUntilIdle => AUTH_UI_TIMEUNTILIDLE
```

Those should both start with `DSPACE_` as well.

Reply:
Updated.